src/indra_cogex/apps/search/search.py

```diff
@@ -66,16 +67,27 @@ def search():
             "mesh_tuple": request.form.get("mesh_tuple"),
         }
         query_params = {k: v for k, v in query_params.items() if v}
+        print(form.mesh_terms.data, request.form.get('mesh_tuple'))
```

Comment:
Do we need the prints for any other purpose than debugging? If not then I think they should be deleted to not bloat the logs when running the service.